marlowe-playground-client/package.json

```diff
@@ -17,15 +17,14 @@
   "dependencies": {
     "@fortawesome/fontawesome-free": "^5.10.2",
     "@popperjs/core": "^2.9.2",
-    "big-integer": "^1.6.50",
     "bignumber": "^1.1.0",
-    "blockly": "^4.20201217.0",
+    "blockly": "^9.3.3",
```

Comment:
Blockly has been heavily refactored since the 4th quarter 2020 release. Among other it is undergoing a TypeScript migration https://github.com/google/blockly/releases